docs/components/movement-sensor/adxl345.md

```diff
@@ -100,37 +143,39 @@ Edit and fill in the attributes as applicable.
-| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [table below](#freefall-attributes).                                                     |
+| `tap`                 | object | Optional     | Holds the configuration values necessary to use the tap detection interrupt on the ADXL345. See [table below](#tap-attributes). |
+| `free_fall`           | object | Optional     | Holds the configuration values necessary to use the free-fall detection interrupt on the ADXL345. See [table below](#freefall-attributes). |
 
 ### Tap attributes
 
 Inside the `tap` object, you can include the following attributes:
 
-| Name                | Type   | Inclusion    | Description                                                                                                                                                                         |
-| ------------------- | ------ | ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `accelerometer_pin` | int    | **Required** | On the accelerometer you can choose to send the interrupts to int1 or int2. Specify this by setting this config value to `1` or `2`.                                                |
-| `interrupt_pin`     | string | **Required** | The `name` of the [digital interrupt](/components/board/#digital_interrupts) you configured for the pin on the [board](/components/board/) wired to the `accelerometer_pin`.        |
-| `exclude_x`         | bool   | Optional     | Tap detection defaults to all three axes. Exclude the x axis by setting this to true. <br> Default: `false`                                                                         |
-| `exclude_y`         | bool   | Optional     | Tap detection defaults to all three axes. Exclude the y axis by setting this to true. <br> Default: `false`                                                                         |
-| `exclude_z`         | bool   | Optional     | Tap detection defaults to all three axes. Exclude the z axis by setting this to true. <br> Default: `false`                                                                         |
-| `threshold`         | float  | Optional     | The magnitude of the threshold value for tap interrupt (in milligrams, between 0 and 15,937). <br> Default: `3000`                                                                  |
+<!-- prettier-ignore -->
+| Name                | Type   | Inclusion    | Description |
+| ------------------- | ------ | ------------ | ----------- |
+| `accelerometer_pin` | int    | **Required** | On the accelerometer you can choose to send the interrupts to int1 or int2. Specify this by setting this config value to `1` or `2`. |
```

Comment:
```suggestion
| `accelerometer_pin` | int    | **Required** | On the accelerometer you can choose to send the interrupts to `int1` or `int2`. Specify this by setting this config value to `1` or `2`. |
```

nit / question: Should these be monospaced?

Reply:
naw because they're breakout board labels not code/JSON values